consul/consul.go

```diff
@@ -90,7 +108,10 @@ func (r *ConsulAdapter) buildCheck(service *bridge.Service) *consulapi.AgentServ
 }
 
 func (r *ConsulAdapter) Deregister(service *bridge.Service) error {
-	return r.client.Agent().ServiceDeregister(service.ID)
+	//	pair := &consulapi.KVPair{Key: "service_attribute" + "/" + service.Name + "/" + k, Value: []byte(v)}
+	success := r.client.Agent().ServiceDeregister(service.ID)
+	r.client.KV().DeleteTree("service_attribute"+"/attributes/"+service.ID, nil)
```

Comment:
Again, shouldn't `"service_attribute"+"/attributes/"` be `r.servicePrefix`?

The test is passing, however that's because you ain't working with the return value for `DeleteTree`. The failure should be at least logged.